tests/end_to_end/dontrun/spark_generic_s3_definitions.py

```diff
@@ -0,0 +1,114 @@
+import os
```

Comment:
Why is this in dontrun

Reply:
didn't remove it when I added it to definitions folder.

Reply:
aight, should remove